flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -634,11 +634,7 @@ function wait_num_of_occurence_in_logs {
     local text=$1
     local number=$2
     local logs
-    if [ -z "$3" ]; then
-        logs="standalonesession"
-    else
-        logs="$3"
-    fi
+    local logs=${3-standalonesession}
```

Comment:
Redundant variable declaration in the previous line.

Reply:
Good catch! It seems something went wrong during the `rebase` op.